docs/concepts/sensor-models/sensor-models.md

```diff
@@ -0,0 +1,88 @@
+
+It is important to understand that a camera model itself does not capture any information related to an image observation. As a purely mathematical representation, a camera model is responsible for mapping the location of a line/sample on the sensor to a location on the target body, and it is completely agnostic of the information that is captured by the sensor.  More information related to image observations and digital numbers can be found [here](@TODO).
+
+Sensor models that support full transformation from the image plane to the BCBF coordinate system fall into two general classes -- rigorous models and rational functional models.  Rigorous sensor models function by taking ephemeris information (position, orientation, and velocity) with respect to some reference frame and utilize mathematical models to spatialize the image location onto the target body [(Laura et al., 2020)](https://doi.org/10.1029/2019EA000713).  Rational functional models approximate this information using rational polynomial coefficients without exposing the interior and exterior orientation of the instrument.  Within this document (and the ASC sensor model ecosystem), we focus on rigorous models due to the availability of interior and exterior orientation information.
+
+
+## The Community Sensor Model
+
+Because the only requirement of a sensor model is that it models a sensor's geometric information, the design and implementation of a sensor model largely remains open-ended.  Of particular importance is the sensor's interface, i.e. what information is required to create the model and what information is produced by the model.  Without a concrete interface specification, each sensor model may require and provide its own set of information, which makes it incredibly difficult to facilitate interoperability between sensor models. In order to address these concerns about interoperability, the US defense and intelligence community established the [Community Sensor Model (CSM) working group](https://gwg.nga.mil/gwg/focus-groups/Community_Sensor_Model_Working_Group_(CSMWG).html) , which is responsible for developing and maintaining a set of standards upon which sensor models can be implemented.  The resulting CSM standard describes an application programming interface (API) for multiple sensor types.
```

Comment:
I am struggling a little here and wonder if other readers might too. The topic sentence is 100% correct. When I read 'the design and implementation of a sensor model', I am not sure whether the sensor model is a mathematical construct, software (it definitely does not read as such), or something physical, like a camera. I wonder if this sentence could be re-written in a way to make it clear that the CSM is computer code that realizes capabilities that one normally wants to accomplish with the mathematical model that is the sensor model?